JitsiConference.js

```diff
@@ -1470,6 +1470,24 @@ JitsiConference.prototype.grantOwner = function(id) {
     this.room.setAffiliation(participant.getJid(), 'owner');
 };
 
+/**
+ * Revoke owner rights to the participant or local Participant as
+ * the user might want to refuse to be a moderator.
+ * @param {string} id id of the participant to revoke owner rights to.
+ */
+JitsiConference.prototype.revokeOwner = function(id) {
+    const participant = this.getParticipantById(id);
+    const isMyself = this.myUserId() === id;
+    const role = this.enableLobby() ? 'member' : 'none';
```

Comment:
This actually enables lobby and returns promise ...

Reply:
You should use this.isMembersOnly()

Reply:
fixed